### ISSUE TITLE
[FEAT] Configure ohwr.org domain

### ISSUE THREAD

**Issue by vascoguita** [OHWR/ohwr.org#171]
## Feature ✨

After migrating the projects from [ohwr.org](ohwr.org), the DNS should be configured to point [ohwr.org](ohwr.org) to GitHub Pages.

## Additional Information ℹ️

Documentation about custom domains and GitHub Pages: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages

**Comment by vascoguita** [comment]
@javier-serrano-pareja, do you know what is our domain registry and who has the account that manages [ohwr.org](ohwr.org)?

**Comment by javier-serrano-pareja** [comment]
@vascoguita the domain is registered with dinahosting.com, with me as the owner. It is paid until 17/04/2029. I think we should take this opportunity to pass the ownership to CERN IT, so they can manage it as they already do for all non-cern.ch domains CERN already owns.

**Comment by vascoguita** [comment]
OK, but maybe we pass it to them after pointing the domain to the new location

**Comment by javier-serrano-pareja** [comment]
Yes, I got all the information from Giacomo, so when the time comes, I know what to do.
